src/data.js

```diff
@@ -3276,6 +3276,28 @@ module.exports = [
       'Ruby on Rails',
     ],
   },
+
+ {
+    name: 'Diyar Faraj',
+    description: 'Developer, programmer, designer, Gym, Soccer ',
+    url: 'https://diyarfaraj.com/uses',
+    twitter: '@diyarfaraj',
+    emoji: '🍕',
+    country: 'SE',
```

Comment:
Try to replace the code `SE` with unicode emoji country code `🇸🇪` and try again.

Reply:
Just did it :)